doc/book/src/dev/platform-tier-policy.md

```diff
@@ -0,0 +1,136 @@
+# Platform Tier Policy
+
+## General
+
+ECC provides three tiers of platform support, modeled after the
+[Rust Target Tier Policy](https://doc.rust-lang.org/stable/rustc/platform-tier-policy.html):
+
+- The Zcash developers provide no guarantees about tier 3 platforms; they exist in the
+  codebase, but may or may not build.
+- ECC's continuous integration checks that tier 2 platforms will always build, but they
+  may or may not pass tests.
+- ECC's continuous integration checks that tier 1 platforms will always build and pass
+  tests.
+
+Adding a new tier 3 platform imposes minimal requirements; we focus primarily on avoiding
+disruption to other ongoing Zcash development.
+
+Tier 2 and tier 1 platforms place work on Zcash developers as a whole, to avoid breaking
+the platform. The broader Zcash community may also feel more inclined to support
+higher-tier platforms in their downstream uses of `zcashd` (though they are not obligated
+to do so). Thus, these tiers require commensurate and ongoing efforts from the maintainers
+of the platform, to demonstrate value and to minimize any disruptions to ongoing Zcash
+development.
+
+This policy defines the requirements for accepting a proposed platform at a given level of
+support.
+
+Each tier builds on all the requirements from the previous tier, unless overridden by a
+stronger requirement.
+
+While these criteria attempt to document the policy, that policy still involves human
+judgment. Platforms must fulfill the spirit of the requirements as well, as determined by
+the judgment of the approving reviewers. Reviewers and team members evaluating platforms
+and platform-specific patches should always use their own best judgment regarding the
+quality of work, and the suitability of a platform for the Zcash project. Neither this
+policy nor any decisions made regarding platforms shall create any binding agreement or
+estoppel by any party.
+
+For a list of all supported platforms and their corresponding tiers ("tier 3", "tier 2",
+or "tier 1"), see [Platform Support](../user/platform-support.md).
+
+The availability or tier of a platform in Zcash releases is not a hard stability guarantee
+about the future availability or tier of that platform. Higher-level platform tiers are an
+increasing commitment to the support of a platform, and we will take that commitment and
+potential disruptions into account when evaluating the potential demotion or removal of a
+platform that has been part of a stable release. The promotion or demotion of a platform
+will not generally affect existing stable releases, only current development and future
+releases.
+
+In this policy, the words "must" and "must not" specify absolute requirements that a
```

Comment:
Use capitalised keywords.

Reply:
Hmm, RFC 2119 is defined in the context of a specification, but this is a policy.